_security-analytics/sec-analytics-config/log-types.md

```diff
@@ -8,22 +8,44 @@ nav_order: 16
+| :--- |:--- |:--- |
+| Access Management | `Ad_ldap` | Active Directory logs that track such things as LDAP queries, errors from the LDAP server, time-out events, and unsecure LDAP binds. |
+| Access Management | `Apache_access` | A log type that is responsible for recording data for all requests processed by an Apache HTTP server. |
+| Access Management | `Okta` | These logs record Okta events from a range of actions such as downloading an export file, requesting an application access, or revoking privileges. |
+| Applications | `GitHub` | Logs that monitor workflows created by [GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions). |
+| Applications| `Gworkspace` | Logs for Google Workspace that can monitor log entries such as admin actions, group and group membership actions, and events having to do with logging in. |  
+| Applications| `M365` | Microsoft 365 audit logs collect a range of data for Microsoft 365 including records from call details, performance data, SQL Server, security events, and access control activity. |
+| Cloud Services | `Azure` | Logs that monitor log data for cloud applications managed by Microsoft Azure Cloud Services. |
+| Cloud Services | `CloudTrail` | Logs that monitor events for an AWS CloudTrail account. OpenSearch can ingest CloudTrail log data from both [AWS Simple Storage Service](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html) (S3) accounts and [Amazon Security Lake](https://docs.aws.amazon.com/security-lake/latest/userguide/what-is-security-lake.html) service accounts. | 
+| Cloud Services | `S3` | These logs track requests for access to an Amazon S3 bucket. |
+| Network Activity| `Dns` | A log that stores Domain Name System (DNS) activity. Security Analytics supports [Open Cybersecurity Schema Framework](https://docs.aws.amazon.com/security-lake/latest/userguide/open-cybersecurity-schema-framework.html) (OCSF) log and event data, which includes the Security Lake log type Route 53. Given that Route 53 is a log that captures DNS activity, its log type should be specified as DNS logs when defining a detector. |
+| Network Activity | `Network` | A log that records events that happen in a system's network, such as login attempts and application events. | 
+| Network Activity | `vpcflow` | [VPC Flow Logs](https://docs.aws.amazon.com/prescriptive-guidance/latest/logging-monitoring-for-application-owners/vpc-flow-logs.html) capture information about the IP traffic going to and from network interfaces in your VPC. |
+| Security | `Waf` |  The role of WAF (Web Application Firewall) is to monitor and filter HTTP traffic between a web application and the internet. It is tasked wth preventing common security attacks such as cross-site scripting (XSS), SQL Injection (SQi), and so on. This new log type (introduced in OpenSearch 2.11) is for users that require monitoring for the WAF use case out-of-the box from the Security plugin. |
```

Comment:
we need to rename almost all the log types and use proper case and acronyms, including updating description with the detailed log types we support